data/notices.yaml

```
@@ -21,7 +21,7 @@
     If you rely on any binaries or libraries that are included in this image beyond the core JRE runtime, consider switching to the [JDK Image](https://images.chainguard.dev/directory/image/jdk/overview).
 
 # Uncomment and edit to add a call to action to the front page of Academy
-# - title: "home"
-#   type: cta
-#   note: |
-#     Ready to secure your Python deployment? Join our <a href="https://events.chainguard.dev/f237831f-f984-45d1-938b-aa4c1faff90a?utm_source=cg-academy-frontpage"> Learning Lab on the Chainguard Image for Python on November 19th </a>!
+- title: "home"
+  type: cta
+  note: |
+    Want to learn about Chainguard's FIPS images? Join our <a href="https://events.chainguard.dev/07bc7b3b-823b-4cd0-a4ab-063d70f2ac93?utm_source=cg-academy&utm_medium=website&utm_campaign=dev-enablement&utm_content=cg-academy-homepage"> Learning Lab on our FIPS Chainguard Images on December 18th </a>!
```

Nit as there is a space between the exclamation point and the end of the last word on the rendered preview

```suggestion
 Want to learn about Chainguard's FIPS images? Join our <a href="https://events.chainguard.dev/07bc7b3b-823b-4cd0-a4ab-063d70f2ac93?utm_source=cg-academy&utm_medium=website&utm_campaign=dev-enablement&utm_content=cg-academy-homepage">Learning Lab on our FIPS Chainguard Images on December 18th</a>!
```